intel/third_party.py

```diff
@@ -16,8 +16,8 @@
 from kubernetes.client.rest import ApiException as K8sApiException
 import datetime
 import logging
-import re
 import time
+from . import util
```

Comment:
If you are using only `ldh_convert_check` from the `util` module, please import it explicitly and not the whole module.